microsoftgraph/microsoft-graph-explorer-v4
# Fix: Add comments as part of the snippets for consistency

## Overview

Fixes #2689 

### Demo
**Before**
![image](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/assets/18407044/9e25fa2a-933e-4183-81f6-f3f2a8a8a837)

**After**
![image](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/assets/18407044/9bdf526d-772a-4067-bf14-ff19af046ad9)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Check out this branch and run GE locally
* Navigate to the code snippets tab
* Select each snippet 
* Observe comments are now part of the code and not separate
